docs/en/index.rst

```diff
@@ -10,90 +10,83 @@
    :caption: TECHNICAL DOCUMENTATION
    :hidden:
 
-   Request-Processing
-   Skill-Attributes
-   Response-Building
-   Alexa-Service-Clients
-   Skill-Builders
-   Managing-In-Skill-Purchases
+   Processing-Request
+   Building-Response
+   Managing-Attributes
+   Calling-Alexa-Service-APIs
+   Configuring-Skill-Instance
 
-===================
+.. toctree::
+   :caption: REFERENCE
+   :hidden:
+
+   TypeDoc <http://ask-sdk-node-typedoc.s3-website-us-east-1.amazonaws.com/>
+
+*******************
 ASK SDK for Node.js
-===================
+*******************
 
-The ASK SDK v2 for Node.js makes it easier for you to build highly
-engaging skills by allowing you to spend more time implementing features
-and less time writing boilerplate code.
+The ASK SDK v2 for Node.js makes it easier for you to build highly engaging skills by allowing you to spend more time implementing features and less time writing boilerplate code.
 
 To help you get started with the SDK, see the following resources.
 
 Guides
```

Comment:
``Guide`` in sync with the toctree caption name?

Reply:
Should be :D

Reply:
Yeah, so it should be updated to ``Guide`` as a heading then